index.html

```diff
@@ -6,13 +6,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify Clone</title>
     <!-- don't forget to link your styles -->
+    <link rel="Stylesheet" href="styles/style.css"/>
   </head>
   <body>
-    Premium Discover Help Download Music for everyone. Spotify is now free on mobile, tablet and computer. Listen to the
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+    <nav> 
+      <img src="/images/spotify-logo.png" alt="Spotify Logo"/>
+      <div>
+        <a>Premium</a>
+        <a href="#discover">Discover</a>
+        <a>Help</a>
+        <a>Download</a>
+      </div>
+    </nav>
+    <section class="landing">
+      <h1>Music for everyone </h1>
+      <h2>Spotify is now free on mobile, tablet and computer. Listen to the right music, wherever you are </h2>
+    </section>
+    <section id="whats-on-spotify">
+      <h3>What’s on Spotify?</h3> 
+      <div>
+        <article class="advantages">
+          <img src="/images/music-icon.png" alt="Music Icon"/>
+          <h4> Millions of Songs </h4>
+          <p>There are millions of songs on Spotify </p>
+        </article>
+        <article class="advantages">
+          <img src="/images/high-quality-icon.png" alt="Quality Icon"/>
+          <h4> HD Music </h4>
+          <p>Listen to music as if you were listening live  </p>
+        </article>
+        <article class="advantages">
+          <img src="/images/devices-icon.png" alt="Devices Icon "/>
```

Comment:
Please add a point before the image route, otherwise it won't be relative, meaning I couldn't see your images in my computer
```html
          <img src="./images/devices-icon.png" alt="Devices Icon "/>
```